{"name": "Verify API handles invalid endpoint", "status": "passed", "steps": [{"name": "Given the API client is initialized for TC_01 Third Scenario", "status": "passed", "start": 1739654846067, "stop": 1739654846067}, {"name": "When I send a GET request to an invalid endpoint", "status": "passed", "start": 1739654846067, "stop": 1739654846684}, {"name": "Then the response status should be 404", "status": "passed", "start": 1739654846684, "stop": 1739654846684}], "start": 1739654846066, "stop": 1739654846684, "uuid": "6d7d8790-f154-4ed4-a7b0-cde5dab6a0d7", "historyId": "ee47dd7b8c3888cdf3c79429f5c48e96", "fullName": "GET User List, User Login and Invalid Endpoint Test: Verify API handles invalid endpoint", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "smoke"}, {"name": "feature", "value": "GET User List, User Login and Invalid Endpoint Test"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}